client/my-sites/upgrades/cart/cart-plan-discount-ad.jsx

```diff
@@ -50,10 +51,35 @@ const CartPlanDiscountAd = React.createClass( {
 			return null;
 		}
 
+		if ( plan.isDomainUpgrade ) {
+			return (
+				<CartAd>
+					<p>{ translate(
+						"You're getting a %(discount)s discount off the regular price of the plan (%(originalPrice)s)" +
+						", because you already paid for the domain.",
+						{
+							args: {
+								discount: plan.formattedDiscount,
+								originalPrice: plan.formattedOriginalPrice
+							}
+						}
+					) }</p>
+					<p>{ translate(
```

Comment:
it looks exactly the same when I omit `<p>` and use `<br>` as separator:

![screen shot 2017-03-08 at 12 33 31](https://cloud.githubusercontent.com/assets/699132/23702420/75f688d2-03fb-11e7-9e4a-52c59c9dae6e.png)